ChessVariantsLogic.Tests/ChessboardTests.cs

```diff
@@ -165,6 +165,41 @@ public void Test_Take()
         Assert.Equal(Constants.UnoccupiedSquareIdentifier,  moveWorker.Board.GetPieceIdentifier("g5"));
         Assert.Equal(Constants.WhiteKnightIdentifier,       moveWorker.Board.GetPieceIdentifier("h7"));
      }
+
+    /// <summary>
+    /// Tests that a bishop, a piece that can not jump, can capture opponents pieces.
+    /// </summary>
+     [Fact]
+     public void PieceNotJumpCanCapture_shouldReturnTrue()
+     {
+        var moveWorker = new MoveWorker(Chessboard.StandardChessboard(), Piece.AllStandardPieces());
+
+        Assert.Equal(Constants.BlackPawnIdentifier, moveWorker.Board.GetPieceIdentifier("b7"));
+
+        Assert.Equal(GameEvent.MoveSucceeded, moveWorker.Move("e2e3"));
+        Assert.Equal(GameEvent.MoveSucceeded, moveWorker.Move("f1a6"));
+        Assert.Equal(GameEvent.MoveSucceeded, moveWorker.Move("a6b7"));
+
+        Assert.Equal(Constants.UnoccupiedSquareIdentifier,  moveWorker.Board.GetPieceIdentifier("a6"));
+        Assert.Equal(Constants.WhiteBishopIdentifier,       moveWorker.Board.GetPieceIdentifier("b7"));
+     }
```

Comment:
The purpose of this test is to make sure that the bishop can capture pieces. Yet, there's asserts on all the steps to set up the test. That isn't necessarily wrong but it couples our tests to the logic to a much higher degree, and we also lose the granularity of the tests. For example, if we update `GameEvent`, we might need to change this test, even though this test doesn't really care about GameEvent. If the moves on lines 179-181 didn't succeed, the asserts on lines 183/184 would fail.

Imo, we should instead only test the outcome we expect. This way, if we introduce a breaking change with `GameEvent`, this test might still pass, allowing us to more easily find the test which showcases the issue.

This applies to more tests, than only this one. Additionally, very many tests creates their own moveWorker (line 175), you can instead create it in the `ChessBoardTests` constructor.

This doesn't necessarily need to be changed, but good to keep in mind in the future. And this is just my opinion, so I'm open to a discussion!